course_content/iris_course/3.Subcube_Extraction.ipynb

```diff
@@ -494,9 +494,9 @@
    "metadata": {},
    "source": [
     "<div class=\"alert alert-block alert-warning\">\n",
-    "    <b><font color='brown'>Exercise: </font></b>Load in the `air_potential_temperature` cube from the file in <pre>iris.sample_data_path('uk_hires.pp')</pre> Iterate over this cube such that you print out subcubes of the following dimensionality:\n",
-    "    <pre>air_potential_temperature / (K)     (time: 3; grid_latitude: 204; grid_longitude: 187)</pre>\n",
-    "    Do this first using `slices` then again using `slices_over`\n",
+    "    <b><font color='brown'>Exercise: </font></b>Load in the `air_potential_temperature` cube from the file in <b><font face=\"courier\" color=\"black\">iris.sample_data_path('uk_hires.pp')</font></b> Iterate over this cube such that you print out subcubes of the following dimensionality:\n",
+    "    <p><b><font face=\"courier\" color=\"black\">air_potential_temperature / (K)     (time: 3; grid_latitude: 204; grid_longitude: 187)</font></b></p>\n",
```

Comment:
This has also lost a linebreak, or at least a full stop, between the highlighted code + "Iterate over..."
( But the "Do this first..." bit still works as it did )